File: C++/ley ohm.cpp
#include <iostream>
using namespace std;

int main() {
    // Variables para almacenar voltaje, resistencia e intensidad
    double Voltaje, Resistencia, Intensidad;

    // Solicitar datos al usuario
    cout << "Ingrese el valor del voltaje del circuito: ";
    cin >> Voltaje;

    cout << "Ingrese el valor de la resistencia del circuito: ";
    cin >> Resistencia;

    // Calcular la intensidad (corriente)
    Intensidad = Voltaje / Resistencia;

    // Mostrar resultado
    cout << "Al conectar un resistor de R " << Resistencia << " ohm a una fuente de V " 
         << Voltaje << " volts, circulará una corriente de " << Intensidad << " amperios." << endl;

    return 0;
}

File: JAVA/ley ohm.java
import java.util.Scanner;

public class LeyDeOhm {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Capturar voltaje y resistencia
        System.out.print("Ingrese el valor del voltaje del circuito: ");
        int voltaje = scanner.nextInt();

        System.out.print("Ingrese el valor de la resistencia del circuito: ");
        int resistencia = scanner.nextInt();

        // Calcular intensidad (amperaje)
        double intensidad = (double) voltaje / resistencia;

        // Mostrar resultado
        System.out.println("Al conectar un resistor de R " + resistencia + " ohm a una fuente de V " 
                            + voltaje + " voltios, circulará una corriente de " + intensidad + " amperios.");

        scanner.close(); // Cerrar el scanner
    }
}
